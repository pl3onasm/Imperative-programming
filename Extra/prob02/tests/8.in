45612377542163945644489634566484332521040883600085458640169945631111074561489561144849456452116354968748465454536989777
1205112345848748551213100001545645615132136854854798546545648748455115444784645411525125416456487489894
