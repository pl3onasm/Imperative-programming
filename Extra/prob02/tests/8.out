45612377542163946849601980415232883734140885145731073772306800485909621107138309599964901236761766493873881911024479671